macros/schema_tests/aggregate_functions/expect_column_proportion_of_unique_values_to_be_between.sql

```diff
@@ -6,7 +6,7 @@
                                                             strictly=False
                                                             ) %}
 {% set expression %}
-count(distinct {{ column_name }})*1.0/count({{ column_name }})
+count(distinct {{ column_name }})*1.000/count({{ column_name }})
```

Comment:
Does Trino not do integer division with `1.0`?

Reply:
In Trino, literal `1.0` has type `decimal(2,1)`. When multiplying integers with it, the result is of decimal type, with the same scale. Therefore, any results are rounded to 1 decimal point in this case. Because of that, [this test](https://github.com/calogica/dbt-expectations/blob/0.10.1/integration_tests/models/schema_tests/schema.yml#L526) is failing, as value with 2 decimal points is specified in the test. Using 3 decimal points should be sufficient for most use cases, I think.

Maybe it would be a cleaner solution to explicitly cast as a float, instead of multiplying by 1.0?
```suggestion
cast(count(distinct {{ column_name }}) as {{ dbt.type_float() }})/count({{ column_name }})
```